docs/docs/retrieval-actions.mdx

```diff
@@ -93,9 +93,10 @@ your responses.
 :::
 
 The retrieval model is trained separately as part of the NLU training pipeline
-to select the correct response. One important thing to remember is that the
-retrieval model uses the text of the response messages to select the correct
-one. If you change the text of these responses, you have to retrain your
+to select the correct response. The default configuration uses the user message text as input and the retrieval intent combined with the
+response key suffix(for e.g. - `chitchat/ask_name`) as the correct label for that user message. However, the
+retrieval model can also be configured to use the text of the response message as the label for training the model.
```

Comment:
how?

Reply:
Added the name of the parameter from the component in the description.